src/classes/RLLP_OppRollup_TEST.cls

```diff
@@ -131,7 +125,6 @@ public with sharing class RLLP_OppRollup_TEST {
         //skip the test if Advancement is installed
         if(ADV_PackageInfo_SVC.useAdv()) return;
 
-    	if (strTestOnly != '*' && strTestOnly != 'testGivingRollupOne2One') return;
```

Comment:
Hi @ceiroa Similar to above. Can you please give me some context about why we need to delete this line? Also, I saw we delete the same line in other methods in this class as well.

Reply:
We don't have to, but these lines in each test method were used just to allow us to run only one method during development by modifying the strTestOnly variable. That was before Salesforce allowed you to choose which methods in a test class to run. They are not needed any more, so I thought I'd clean them up.